src/WorkflowManager/WorkflowManager/Validators/WorkflowValidator.cs

```diff
@@ -194,17 +213,27 @@ private static void ValidateTask(TaskObject[] tasks, TaskObject currentTask, int
                 paths = new List<string>();
             }
 
-            if (currentTask.Artifacts != null && !currentTask.Artifacts.Output.IsNullOrEmpty())
+            if (currentTask.Artifacts != null && currentTask.Artifacts.Output.IsNullOrEmpty() is false)
             {
                 var uniqueOutputNames = new HashSet<string>();
                 var allOutputsUnique = currentTask.Artifacts.Output.All(x => uniqueOutputNames.Add(x.Name));
 
-                if (!allOutputsUnique)
+                if (allOutputsUnique is false)
                 {
                     Errors.Add($"Task: \"{currentTask.Id}\" has multiple output names with the same value.\n");
                 }
             }
 
+            if (currentTask.Type.Equals("argo", StringComparison.OrdinalIgnoreCase) is true)
+            {
+                ValidateArgoTask(currentTask);
+            }
+
+            if (currentTask.Type.Equals("clinical-review", StringComparison.OrdinalIgnoreCase) is true)
```

Comment:
```suggestion
            if (currentTask.Type.Equals("clinical-review", StringComparison.OrdinalIgnoreCase))
```

Could probably simplify it